commands/Information/urbandictionary.js

```diff
@@ -0,0 +1,103 @@
+const { Command, RichDisplay } = require('klasa');
+const { MessageEmbed } = require('discord.js');
+const fetch = require('node-fetch');
+
+module.exports = class extends Command {
+
+    constructor(...args) {
+        /**
+         * Any default options can be omitted completely.
+         * if all options are default, you can omit the constructor completely
+         */
+        super(...args, {
+            name: "define",
+            enabled: true,
+            runIn: ['text', 'dm', 'group'],
+            requiredPermissions: [],
+            requiredSettings: [],
+            aliases: ["urbandictionary"],
+            autoAliases: true,
+            bucket: 1,
+            cooldown: 5,
+            promptLimit: 0,
+            promptTime: 30000,
+            deletable: false,
+            guarded: false,
+            nsfw: false,
+            permissionLevel: 0,
+            description: 'Looks up facts on Urban Dictonary.',
+            extendedHelp: '',
+            usage: '<searchterm:string>',
+            usageDelim: undefined,
+            quotedStringSupport: false,
+            subcommands: false
+        });
+    }
+
+    async run(message, [searchterm]) {
+        try {
```

Comment:
Consider changing the try catch to only needed areas. Wrapping it around the whole code is considered bad practice.